engine/flink/process/src/main/scala/pl/touk/nussknacker/engine/process/util/StateConfiguration.scala

```diff
@@ -15,7 +14,9 @@ object StateConfiguration {
 
   }
 
-  case class RocksDBStateBackendConfig(checkpointDataUri: String,
+  case class RocksDBStateBackendConfig(enable: Boolean,
```

Comment:
maybe let's make true default? by Option or default param?

Reply:
After discussion - we will left it as it is.